src/bosh_azure_cpi/lib/cloud/azure/utils/command_runner.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Bosh::AzureCloud
+  class CommandRunner
+    include Helpers
+
+    def initialize
+      @logger = Bosh::Clouds::Config.logger
+    end
+
+    def run_command(command, log_cmd = true)
+      @logger.info(command) if log_cmd
+      output, status = Open3.capture2e(command)
+      cloud_error("'#{command}' failed with exit status=#{status.exitstatus} [#{output}]") if status.exitstatus != 0
```

Comment:
Too heavy to include Helpers here. And recommended not to use cloud_error here...  `raise` would be better.

Reply:
yes, indeed.